src/app.js

```diff
@@ -1 +1,52 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+console.log("Hallo daar");
+
+const countryCurrency = (currencies) => {
+    let localC = "";
+    for (let i = 0; i < currencies.length; i++) {
+        if (i > 0) {
+            localC += ' and ' + currencies[i].name + 's. ';
+        } else {
+            localC = currencies[i].name;
+        }
+    }
+    console.log(localC);
+    return localC
+}
+
+
+async function getCountryInformation() {
+    const BASE = "https://restcountries.com/";
+    const ENDPOINT = "v2/name/";
+
+    const searchField = document.getElementById('search-field');
+    const searchFieldOutput = searchField.value
+    searchField.value = '';
+
+    try {
+        const response = await axios.get(BASE + ENDPOINT + searchFieldOutput);
+        let countryOutput = response.data
+
+        const listOfCountries = document.getElementById('country-list')
+        countryOutput.map((country) =>
```

Comment:
Hier zou je nog spreading kunnen toepassen, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
        countryOutput.map(({ name, population, currencies, capital, region, flag: { png } }) =>
```